lib/runtime-c-api/Cargo.toml

```diff
@@ -15,16 +15,21 @@ crate-type = ["cdylib", "rlib", "staticlib"]
 libc = "0.2.60"
 
 [dependencies.wasmer-runtime]
+default-features = false
 path = "../runtime"
 version = "0.6.0"
 
 [dependencies.wasmer-runtime-core]
+default-features = false
 path = "../runtime-core"
 version = "0.6.0"
 
 [features]
+default = ["cranelift-backend"]
 debug = ["wasmer-runtime/debug"]
-llvm = ["wasmer-runtime/llvm"]
+cranelift-backend = ["wasmer-runtime/cranelift", "wasmer-runtime/default-backend-cranelift"]
+llvm-backend = ["wasmer-runtime/llvm"]
```

Comment:
Why not adding `default-backend-llvm` to the list of features to enable with `llvm-backend`?